presto-main/src/main/java/io/prestosql/sql/analyzer/FeaturesConfig.java

```diff
@@ -581,6 +582,18 @@ public FeaturesConfig setEnableStatsCalculator(boolean enableStatsCalculator)
         return this;
     }
 
+    public boolean isCollectPlanStatistics()
+    {
+        return collectPlanStatistics;
+    }
+
+    @Config("experimental.collect-plan-statistics")
```

Comment:
The new behavior is different than the one we used to have.
So why experimental?

Reply:
Because `experimental.enable-stats-calculator` is experimental itself.

Reply:
That's a mistake. If something is enabled by default, it should be considered "experimental".
Or, more precisely, if something is experimental, it should not be enabled by default.

Reply:
https://github.com/prestosql/presto/pull/1875, https://github.com/airlift/airlift/pull/767